components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3906,6 +3907,11 @@ void RewardsServiceImpl::OnCompleteReset(
     const bool success) {
   resetting_rewards_ = false;
 
+  auto* ads_service = brave_ads::AdsServiceFactory::GetForProfile(profile_);
```

Comment:
not sure if this block is needed as you already have the same call in `RewardsServiceImpl::CompleteReset`. Line 3889

Reply:
Fixed